packages/expo-modules-core/src/index.ts

```diff
@@ -9,6 +9,7 @@ import { CodedError } from './errors/CodedError';
 import { UnavailabilityError } from './errors/UnavailabilityError';
 
 import './sweet/setUpErrorManager.fx';
+import './web/index';
```

Comment:
We don't want web-specific code to be included in bundles for other platforms. I know there is `index.web.ts`, but will this work on other platforms? I guess Metro would say it's unable to resolve the file path.

Reply:
I tested it out and it worked on iOS – was surprised too.
If this is confusing I can add an empty `index.ts` – wdyt?